### PR TITLE
fix(properties): replaced YML settings title

### LINKED ISSUE #28: Settings name
This extension settings have a weird category name. See picture…
exampleconf
“Exampe configuration” is not a very good name…

### PR DESCRIPTION
**Pull request by alacambre-yseop**
Fix #28

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "contributes": {
         "configuration": {
             "type": "object",
-            "title": "Example configuration",
+            "title": "Yseop Markup Language Configuration",
```
Comment (mminot-yseop) [inline client/package.json]: Use the same capitalization rules as other configs. → “configuration” (OK for Yseop Markup Language because YML, though)

Comment (alacambre-yseop) [reply client/package.json]: Done.

**Comment by mminot-yseop** [comment]
That wasn't a “typo” _per se_, but OK.

**Comment by mminot-yseop** [comment]
Squashed because it seemed better for this particular thing.

**Pull request merged**